web/auth/oauth.go

```diff
@@ -877,6 +884,11 @@ func accessToken(c echo.Context) error {
 				"error": "invalid refresh token",
 			})
 		}
+
+		// Update the last_refreshed_at field of the OAuth client
+		client.LastRefreshedAt = time.Now()
```

Comment:
Could this be added when the token is first created as well? So we can always rely on this field even though it was not  refreshed yet

Reply:
We have `cozyMetadata.created_at`. It is not enough?

Reply:
We can use this, but the client might not be aware that the `LastRefreshedAt` won't be there before a week, so I feel like it could prevent future mistakes

Reply:
OK, if you prefer, I add it.